code/game/atoms.dm

```diff
@@ -1431,3 +1431,7 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
  */
 /atom/proc/relaydrive(mob/living/user, direction)
 	return !(SEND_SIGNAL(src, COMSIG_RIDDEN_DRIVER_MOVE, user, direction) & COMPONENT_DRIVER_BLOCK_MOVE)
+
+/// Used with the spawner component to do something when a mob is spawned.
+/atom/proc/on_mob_spawn(atom/created_atom)
```

Comment:
Wouldn't it make more sense for the argument to be some form of `/mob`?

Reply:
changed